pliers/extractors/audio.py

```diff
@@ -618,3 +618,54 @@ def _extract(self, stim):
         return ExtractorResult(preds, stim, self, features=labels,
                                onsets=onsets, durations=[dur]*len(onsets),
                                orders=list(range(len(onsets))))
+
+
+class TimbrePitchExtractor(AudioExtractor):
```

Comment:
Let's go ahead and rename this extractor something like `MFCCEnergyExtractor`

Darius, suggestions are welcome

Reply:
At some point we might need/want to move all of the librosa-based classes into their own module and/or rename them all to make the extractor source explicit. But we can hold off until it becomes an issue (e.g., if someone wants to use a different tool to extract a similar feature).